runtime/src/bank.rs

```diff
@@ -347,6 +347,8 @@ pub struct Bank {
     /// Rewards that were paid out immediately after this bank was created
     #[serde(skip)]
     pub rewards: Option<Vec<(Pubkey, i64)>>,
+
+    accounts_hash_epoch: u64,
```

Comment:
Can adding this field be avoided?  Doing so will break snapshots

Reply:
Hmm. I'll think about it.